simple/msg_pingpong.c

```diff
@@ -552,15 +548,33 @@ static int run(void)
 	return ret;
 }
 
+void print_usage(char *name)
+{
+	fprintf(stderr, "Usage:\n");
+	fprintf(stderr, "  %s [OPTIONS]\t\tstart ping server\n", name);
+	fprintf(stderr, "  %s [OPTIONS] <host>\tpong given host\n", name);
+
+	fprintf(stderr, "\nOptions:\n");
+	fprintf(stderr, "  -n\tdomain_name\n");
+	fprintf(stderr, "  -p\tport_number\n");
+	fprintf(stderr, "  -s\tsource_address\n");
+	fprintf(stderr, "  -I\tnumber of iterations\n");
+	fprintf(stderr, "  -S\tspecific transfer size or 'all'\n");
+	fprintf(stderr, "  -m\tmachine readable output\n");
+	fprintf(stderr, "  -i\tprint hints structure and exit\n");
+	fprintf(stderr, "  -v\tdisplay versions and exit\n");
+	fprintf(stderr, "  -h\tdisplay this help output\n");
+
+	return;
+}
+
 int main(int argc, char **argv)
 {
-	int op;
+	int op, ret;
+	int prhints = 0;
 
-	while ((op = getopt(argc, argv, "d:n:p:s:I:S:m")) != -1) {
+	while ((op = getopt(argc, argv, "n:p:s:I:S:mivh")) != -1) {
 		switch (op) {
-		case 'd':
```

Comment:
My desireis that all client/server tests share have these base options (and look like this):

```
./simple/fi_msg_pingpong -h
Usage:
  ./simple/fi_msg_pingpong [OPTIONS]            start ping server
  ./simple/fi_msg_pingpong [OPTIONS] <host>     pong given host

Options:
  -n    domain_name
  -p    port_number
  -s    source_address
  -I    number of iterations
  -S    specific transfer size or 'all'
  -m    machine readable output
  -i    print hints structure and exit
  -v    display versions and exit
  -h    display this help output
```